azure-pipelines/PoliCheckExclusions.xml

```diff
@@ -6,5 +6,5 @@
     <!--Each of these file types will be completely skipped for the entire scan -->
     <!-- <Exclusion Type="FileType">.ABC|.XYZ</Exclusion>-->
     <!--The specified file names will be skipped during the scan regardless which folder they are in -->
-    <!-- <Exclusion Type="FileName">ABC.TXT|XYZ.CS</Exclusion>-->
+    <Exclusion Type="FileName">LICENSE.TXT</Exclusion>
```

Comment:
Where is there a `LICENSE.TXT` file? Not in this repo, unless it's under the yarn cache somewhere.

Reply:
src/servicebroker-npm/LICENSE.txt